doc/api/next_api_changes/behaviour.rst

```diff
@@ -50,3 +50,9 @@ and `.pyplot.yticks` would result in
     TypeError: object of type 'NoneType' has no len()
 
 It now raises a ``TypeError`` with a proper description of the error.
+
+Setting the same property under multiple aliases now raises a TypeError
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, calling e.g. ``plot(..., color=somecolor, c=somecolor)`` would
```

Comment:
AFAICS, there was no special handling if the parameter values are the same. So would do
```suggestion
Previously, calling e.g. ``plot(..., color=somecolor, c=othercolor)`` would
```

Reply:
done